File: Module 1/ConditionalStructure_2/Exercise/BodyMassIndex/BodyMassIndex.js
let inputWeight = prompt('Enter your weight (kg): ');
let inputHeight = prompt('Enter your height (m): ');

let weight = Number(inputWeight);
let height = Number(inputHeight);

let bmi = weight / ( height ^ 2 );

if (bmi < 18)
    document.write("Underweight");
else if (bmi < 25.0)
    document.write("Normal");
else if (bmi < 30.0)
    document.write("Overweight");
else
    document.write("Obese");

File: Module 2/src/_01_Introduction_to_Java/Practice/BodyMassIndex.java
package _01_Introduction_to_Java.Practice;

import java.sql.SQLOutput;
import java.util.Scanner;

public class BodyMassIndex {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter your height (m): ");
        float height = scanner.nextFloat();
        System.out.print("Enter your weight (kg): ");
        float weight = scanner.nextFloat();

        float bmi = (float) (weight / (Math.pow(height, 2)));

        if (bmi < 18)
            System.out.println("Underweight");
        else if (bmi < 25.0)
            System.out.println("Normal");
        else if (bmi < 30.0)
            System.out.println("Overweight");
        else
            System.out.println("Obese");
    }
}
